src/Functions.Worker.Extensions.DurableTask.SqlServer/Functions.Worker.Extensions.DurableTask.SqlServer.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <!-- Import common properties for all shipping packages-->
+  <Import Project="../common.props" />
+  
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+  </PropertyGroup>
+  
+  <!-- NuGet package settings -->
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Functions.Worker.Extensions.DurableTask.SqlServer</PackageId>
+    <Title>Azure Durable Functions SQL Provider</Title>
+    <Description>Microsoft SQL provider for Azure Durable Functions.</Description>
+    <PackageTags>Microsoft;Azure;Functions;Durable;Task;Orchestration;Workflow;Activity;Reliable;SQL</PackageTags>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Abstractions" Version="1.1.0" />
```

Comment:
@jviau: can we also add a comment here to update the AssemblyInfo file when this gets changed?

Reply:
This is not the version that is to be kept in sync. It is the version of Microsoft.Azure.DurableTask.SqlServer.AzureFunctions. Ultimately, it would be nice to automate this - but that can be left for a separate PR as we would need to extract some common versioning targets between the two projects.

Reply:
Ah, my mistake. I misread this (twice, one per PR). Thanks, this makes sense.